File: bachngocsach/src/detail.lua
local url = ...

local doc = http:request(url):html()

if doc ~= nil then
    local book = {}
    book["name"] =  doc:select("h1#truyen-title"):text()
    book["cover"] = core:merge_url("http://bachngocsach.com", doc:select("div#anhbia img"):attr("src"))
    book["author"] = doc:select("div#tacgia a"):text()
    book["description"] = doc:select("div#gioithieu"):html()
    book["detail"] = doc:select("div#tacgia"):html() .. "<br>" .. doc:select("div#theloai"):html()
    return book
end
return nil

File: saytruyen/detail.js
function execute(url) {
    const doc = Http.get(url).html();

    return Response.success({
        name: doc.select("h1.hl-name-book").first().text(),
        cover: doc.select(".book-thum img").first().attr("src"),
        author: doc.select("a[href~=truyen-tac-gia]").first().text(),
        description: doc.select(".box-show-des").html(),
        detail: doc.select(".book-list-field .l-book-list-field").html(),
        host: "https://saytruyen.com",
        ongoing: doc.select(".status-chapter").text().indexOf("Đang ra") >= 0
    });
}